README.md

```diff
@@ -157,9 +157,9 @@ When you want to resume your project, you first need to create a connection to t
 
 Finally, go to the main page (house icon) and click Open Cloud Project. Then select the blob storage connection, and select your project's .proj file. The application will load all of the project's settings because it has the security token.
 
-#### Keyboard Shortcuts ####
+#### Keyboard Shortcuts and useful tips ####
 
-Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster.
+Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster. You can find them in the right top corner (icon with book and letter "i" ![alt text](docs/images/keyboard-shortcuts-icon.png "Keyboard shortcuts menu icon")).
```

Comment:
The image isn't showing up for me in rich diff

Reply:
For me either. But it does on the branch itself https://github.com/microsoft/OCR-Form-Tools/tree/alex-krasn/docs-update-README-keyboard-shortcuts

Reply:
Can we move the image to a new line? 
![image](https://user-images.githubusercontent.com/60453211/84195701-dd4c6400-aa53-11ea-9549-d018b2f2343f.png)

maybe something like this:
```
You can view them by clicking the following icon on the right side of the title bar:
![alt text](docs/images/keyboard-shortcuts-icon.png "Keyboard shortcuts menu icon")
```